### PR TITLE
Release announcement for cargo-audit v0.18

### PR DESCRIPTION
**Pull request by Shnatsel**

### CODE REVIEW

**Comment by thomcc** [comment]
This is a third party crate, no? It's a great tool, but I'm not sure why the update would be discussed on this blog.

**Comment by Shnatsel** [comment]
This blog is also meant for Rust WGs to publish updates on their work, according to the README. This tool is maintained by an official WG.

Previous updates on this tool have also been published on this blog, e.g. https://github.com/rust-lang/blog.rust-lang.org/blob/master/posts/inside-rust/2019-10-03-Keeping-secure-with-cargo-audit-0.9.md

**Comment by thomcc** [comment]
Hm, fair enough.

**Comment by Shnatsel** [comment]
I see this has not been reviewed/merged in 2 days. I don't have the necessary permissions on this repo to assign reviewers. Is there anyone in particular I should reach out to in order to get this reviewed and merged?

File: posts/2023-08-29-keeping-secure-with-cargo-audit-0.18.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+[`cargo audit`](https://crates.io/crates/cargo-audit) checks your project's dependencies for known security vulnerabilites.
+
+By default `cargo audit` checks on your `Cargo.lock` file, but it can also [scan compiled binaries](https://github.com/rustsec/rustsec/tree/main/cargo-audit#cargo-audit-bin-subcommand). You can install `cargo-audit` and run it against your project with the following commands:
+
+```
+$ cargo install cargo-audit
+$ cargo audit
+```
+
+Both `cargo audit` and the [RustSec](https://rustsec.org/) advisory database that powers it are maintaiend by the [Rust Secure Code working group](https://www.rust-lang.org/governance/wgs/wg-secure-code).
```
Comment (calebzulawski) [inline posts/2023-08-29-keeping-secure-with-cargo-audit-0.18.md:18]: Typo

**Review by calebzulawski — CHANGES_REQUESTED** [review]

**Review by calebzulawski — APPROVED** [review]

**Comment by calebzulawski** [comment]
Apparently I could fix that typo from the mobile website :) Looks good to me, I'll leave the merge up to you

**Comment by Shnatsel** [comment]
Thanks for the quick review!

**Comment by Shnatsel** [comment]
@thomcc I moved the post to inside-rust. It was meant to go there, I put it on the main blog by mistake.

**Comment by Shnatsel** [comment]
I need this to go live, but I don't have permission to merge it.

Could someone please push the button? :sweat_smile:

**Comment by Shnatsel** [comment]
Thank you! :heart:

**Pull request merged**